WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeListActivity.java

```diff
@@ -282,10 +289,16 @@ public void onEventMainThread(PublicizeEvents.ActionRequestChooseAccount event)
 
         closeWebViewFragment();
 
+        SiteModel site = mSiteStore.getSiteBySiteId(event.getSiteId());
```

Comment:
I can't think of a case where the site wouldn't be in the store, but I decided to add that `null` check just to be safe. I think we're fine just showing that toast and leaving the routine.